frontend/public/locales/en/translation.json

```diff
@@ -2115,7 +2116,8 @@
     "submariner.install.form.gateways.labelHelp": "The number of worker nodes  that will be used to deploy the Submariner gateway component on the managed cluster. If the value is greater than 1, the Submariner gateway HA will be enabled automatically. (default 1)",
     "submariner.install.form.gateways.placeholder": "Enter gateway count",
     "submariner.install.form.instancetype": "Instance type",
-    "submariner.install.form.instancetype.labelHelp": "The Amazon Web Services EC2 instance type of the gateway node that will be created on the managed cluster. (default c5d.large)",
+    "submariner.install.form.instancetype.labelHelp.aws": "The Amazon Web Services EC2 instance type of the gateway node that will be created on the managed cluster. (default c5d.large)",
+    "submariner.install.form.instancetype.labelHelp.azure": "The Azure instance type of the gateway node that will be created on the managed cluster. (default Standard_F4s_v2)",
     "submariner.install.form.instancetype.placeholder": "Select instance type",
     "submariner.install.form.nattenable": "Enable NAT-T",
     "submariner.install.form.nattenable.labelHelp": "Submariner can use NAT Traversal for IPsec tunnels between clusters. Set this to true to enable NAT Traversal and false otherwise (default true)",
```

Comment:
Change to:

Set the value to true to enable NAT Traversal and false to disable NAT Traversal (default true).

Reply:
Done